dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05205UpdateIndexNameLengthTest.java

```diff
@@ -0,0 +1,97 @@
+package com.dotmarketing.startup.runonce;
+
+import static org.junit.Assert.assertEquals;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.SQLException;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task05205UpdateIndexNameLengthTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    private Map<String, String> getColumnProperties() throws DotDataException {
+        String query;
+
+        if (!DbConnectionFactory.getDBType().equals("Oracle")){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "Oracle" appears 4 times in this file; the first occurrence is on line 25](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)